bridge/client/app/app.component.ts

```diff
@@ -11,20 +13,20 @@ declare let dT_: any;
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
-export class AppComponent implements OnInit {
-  constructor(private http: HttpClient, private dataService: DataService) {
+export class AppComponent {
+  constructor(private http: HttpClient, private dataService: DataService, private route: ActivatedRoute) {
     if (typeof dT_ !== 'undefined' && dT_.initAngularNg) {
       dT_.initAngularNg(http, HttpHeaders);
     }
-  }
-
-  public ngOnInit(): void {
     this.dataService.loadKeptnInfo();
-    this.dataService.keptnInfo
-      .pipe(filter((keptnInfo) => !!keptnInfo))
+    const keptnInfo$ = this.dataService.keptnInfo.pipe(filter((keptnInfo) => !!keptnInfo));
+
+    combineLatest([this.route.firstChild?.data ?? of({}), keptnInfo$])
       .pipe(take(1))
-      .subscribe(() => {
-        this.dataService.loadProjects();
+      .subscribe(([data]) => {
+        if (!data.projectsHandledByComponent) {
+          this.dataService.loadProjects().subscribe();
+        }
       });
```

Comment:
Suggestion: Get rid of double subscription
```
 combineLatest([this.route.firstChild?.data ?? of({}), keptnInfo$]).pipe(
      take(1),
      filter(([data]) => !data.projectsHandledByComponent),
      mergeMap(() => this.dataService.loadProjects()),
).subscribe();

```

Reply:
The reason why I put the check inside the subscribe instead of the pipe was to only trigger it once, as we don't care for any changes here.
If the filter is moved into the pipe, there would be another projects call if you redirect from the dashboard to another view.